src/components/network-map-tab.js

```diff
@@ -561,15 +561,28 @@ export const NetworkMapTab = ({
 
         dispatch(resetMapReloaded());
 
+        const previousCurrentNode = currentNode;
+
         return mapEquipments
             .reloadImpactedSubstationsEquipments(
                 studyUuid,
                 currentNode,
-                updatedSubstationsToSend,
-                setUpdatedLines
+                updatedSubstationsToSend
             )
-            .finally(() => {
-                setWaitingLoadData(false);
+            .then((results) => {
+                const isFullReload = updatedSubstationsToSend ? false : true;
```

Comment:
You can move this inside the if

Reply:
Done

Reply:
you could use : 
mapEquipments.updateSubstations(
                        mapEquipments.checkAndGetValues(results[0]),
                        !!updatedSubstationsToSend
                    );